### PR TITLE
Update ecflow after switching blending to MPMD parallel run.

### PR DESCRIPTION
**Pull request by YaliMao-NOAA**
Update ecflow after switching blending to MPMD parallel run.
 1. Don't need to setup ecflow links for blending
 2. In ecflow definations, change event triggers of each forecast hour to f048 of the upstream completion
 
Change COMROOT from 'com' to '%ENVIR%/com'

### CODE REVIEW

File: ecf/scripts/grib2/0p25/blending/jwafs_grib2_0p25_blending.ecf [hunk 1]
```diff
@@ -30,6 +29,9 @@ module load libjpeg/${libjpeg_ver}
 module load grib_util/${grib_util_ver}
 module load util_shared/${util_shared_ver}
 
+module load cray-pals/$craypals_ver
+module load cfp/$cfp_ver
+
```
Comment (aerorahul) [inline ecf/scripts/grib2/0p25/blending/jwafs_grib2_0p25_blending.ecf]:
```suggestion
module load cray-pals/${craypals_ver}
module load cfp/${cfp_ver}

```

File: ecf/scripts/grib2/0p25/blending/jwafs_grib2_0p25_blending.ecf
Comment (aerorahul) [inline ecf/scripts/grib2/0p25/blending/jwafs_grib2_0p25_blending.ecf:1]: Lines 46-48.
Why should a SPA get notified if `envir != prod`?
Line 49
`ncep.sos@noaa.gov` is repeated.

**Review by aerorahul — CHANGES_REQUESTED** [review]
Please remove the line:
```
ecf/scripts/grib2/0p25/blending/jwafs_grib2_0p25_blending_f*.ecf
```
from `.gitignore`.
Please see comments inline.

File: ecf/scripts/grib2/0p25/blending/jwafs_grib2_0p25_blending.ecf
Comment (YaliMao-NOAA) [reply ecf/scripts/grib2/0p25/blending/jwafs_grib2_0p25_blending.ecf:1]: Line 46-48 was done by NCO so I can't answer this question.
Line 49 has two 'sos' emails, one is ncep, the other is nco. Those email were also provided by NCO.

**Review by aerorahul — APPROVED** [review]
looks good.

**Pull request merged**